src/components/common/DetailComment.vue

```diff
@@ -36,7 +36,7 @@ export default class DetailComment extends Vue {
   get rules (): Array<(val) => boolean | string> {
     // include whitespace in maximum length check
     return [
-      val => (val && val.trim().length > 0) || 'Detail is required.',
+      val => (val && val.trim().length > 0) || this.textRequiredErrorMsg,
```

Comment:
Using this new prop to show "Reason is required" instead of "Detail is required"
![reason is required](https://github.com/JazzarKarim/business-filings-ui/assets/122301442/1ddf5e75-8bba-4108-ad31-1936ab319b4a)

Reply:
To make this component more generic?

Reply:
Yup, exactly.

Reply:
Great, thanks. No further questions on this.